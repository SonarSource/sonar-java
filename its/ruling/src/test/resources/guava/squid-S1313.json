{
'com.google.guava:guava:src/com/google/common/net/InetAddresses.java':[
118,
119,
],
}
