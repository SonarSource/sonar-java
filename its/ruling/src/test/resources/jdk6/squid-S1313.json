{
'jdk6:java/security/Signature.java':[
455,
],
'jdk6:java/security/cert/X509CRLSelector.java':[
604,
],
'jdk6:java/security/cert/X509CertSelector.java':[
115,
116,
117,
118,
119,
2094,
2128,
],
}
