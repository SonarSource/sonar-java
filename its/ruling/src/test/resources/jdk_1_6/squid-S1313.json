{
'jdk_1_6:java/net/Inet4AddressImpl.java':[
25,
],
'jdk_1_6:java/security/Signature.java':[
455,
],
'jdk_1_6:java/security/cert/X509CRLSelector.java':[
604,
],
'jdk_1_6:java/security/cert/X509CertSelector.java':[
115,
116,
117,
118,
119,
2094,
2128,
],
}
