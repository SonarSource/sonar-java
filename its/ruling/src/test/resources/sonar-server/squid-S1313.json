{
'org.sonarsource.sonarqube:sonar-server:src/main/java/org/sonar/server/serverid/ws/GenerateAction.java':[
74,
],
}
