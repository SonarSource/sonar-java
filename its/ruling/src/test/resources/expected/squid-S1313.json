{
'project:com/sun/corba/se/impl/orb/DataCollectorBase.java':[
154,
156,
],
'project:com/sun/jmx/snmp/IPAcl/PrincipalImpl.java':[
66,
],
'project:com/sun/org/apache/xml/internal/security/keys/content/x509/XMLX509SKI.java':[
59,
],
'project:java/net/Inet4AddressImpl.java':[
43,
],
'project:java/security/Signature.java':[
486,
],
'project:java/security/cert/CertificateRevokedException.java':[
146,
],
'project:java/security/cert/X509CRLSelector.java':[
623,
],
'project:java/security/cert/X509CertSelector.java':[
133,
134,
135,
136,
137,
2109,
2143,
],
}
